.github/workflows/ci_win.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
   test_env:
     name: test_environments
-    runs-on: "ubuntu-latest"
+    runs-on: "windows-latest"
```

Comment:
things run faster on ubunutu

Reply:
Yup, and it is also tested in `ci.yml`, though this one only runs one test `-k environment_bench` and is meant to check that environment support (`conda`, `mamba`, `virtualenv`) works on `windows` so it should stay as-is, since the other windows tests are only run with `virtualenv`.

Reply:
OK